includes/class-wc-stripe-checkout-session-manager.php

```
@@ -0,0 +1,341 @@
+					[ 'line_items' => $this->build_line_items( $cart_context ) ],
+					"checkout/sessions/$session_id"
+				);
+
+				if ( ! empty( $checkout_session->error ) ) {
+					$message = empty( $checkout_session->error->message ) ? __( 'Checkout Sessions update API returned an error', 'woocommerce-gateway-stripe' ) : $checkout_session->error->message;
+					throw new Exception( $message );
+				}
+
+				WC_Stripe_Checkout_Session_Context::store_for_cart( $session_id, $cart_context );
+			}
+		);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Checkout_Session_Context' in method 'update_session'. (undefined)

(StaticAccess)